x-pack/plugins/synthetics/server/synthetics_service/project_monitor_formatter.ts

```diff
@@ -0,0 +1,302 @@
+/*
```

Comment:
I appreciate the detailed REST API tests. Will be nice to add some basic unit tests for this class.